bundler/lib/bundler/cli/init.rb

```diff
@@ -32,20 +32,36 @@ def run
           file << spec.to_gemfile
         end
       else
-        File.open(File.expand_path("../templates/#{gemfile}", __dir__), "r") do |template|
+        File.open(File.expand_path(template_path, __dir__), "r") do |template|
           File.open(gemfile, "wb") do |destination|
             IO.copy_stream(template, destination)
           end
         end
       end
 
-      puts "Writing new #{gemfile} to #{SharedHelpers.pwd}/#{gemfile}"
+      print_feedback
     end
 
     private
 
     def gemfile
       @gemfile ||= Bundler.preferred_gemfile_name
     end
+
+    def template_path
+      template_folder = "../templates/"
+      if File.exist?(template_folder + gemfile)
+        template_folder + gemfile
+      else
+        template_folder + "Gemfile"
+      end
+    end
+
+    def print_feedback
+      # We need this because when the key BUNDLE_GEMFILE is seted #gemfile will
```

Comment:
:scissors: `s/seted/set/g` :question:

Reply:
I don't understand, could you explain in another way, please?

Reply:
Sorry, I'm still learning English (my main language is Portuguese), I'll correct it.